ocaml/idl/datamodel_types.ml

```diff
@@ -267,13 +267,13 @@ let release_order_full = [{
      code_name     = Some rel_stockholm;
      version_major = 2;
      version_minor = 15;
-     branding      = "Unreleased";
-     release_date  = None;
+     branding      = "Citrix Hypervisor 8.2";
+     release_date  = Some "June 2020";
    }; {
-     code_name     = Some rel_next;
+     code_name     = Some rel_stockholm_psr;
      version_major = 2;
-     version_minor = 16;
-     branding      = "Unreleased";
+     version_minor = 15;
+     branding      = "Citrix Hypervisor 8.2 Hotfix n";
```

Comment:
I assume this is a placeholder. Just in the case it does leak out, I'd remove the trailing ` n` or keep the "Unreleased". Technically the string does not matter, only the version number.

Reply:
This actually introduces a conflict with 1.249-lcm because of https://github.com/xapi-project/xen-api/pull/4168, so we'll have an extra to nudge here to make sure we remove this placeholder